.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,31 @@
+name: reviewdog
+
+on: [push, pull_request]
+
+jobs:
+  misspell:
+    name: runner / misspell
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code.
+        uses: actions/checkout@v3
+      - uses: haya14busa/action-cond@v1
+        id: reporter
+        with:
+          cond: ${{ github.event_name == 'pull_request' }}
+          if_true: "github-pr-review"
+          if_false: "github-check"
+      - name: Generate list of generated files
+        uses: gh640/command-result-action@v1  # see https://gotohayato.com/content/558/
```

Comment:
Please remove the link (a Japanese website would not be useful for non-native speakeers...)

Reply:
Sorry, remove it.